docs/crud/property.md

```diff
@@ -111,6 +128,45 @@ List operation lists all properties in a group with a name.
 $ bydbctl property list -g sw -n ui_template
 ```
 
+## TTL field in a property
+
+TTL field in a property is used to set the time to live of the property. The property will be deleted automatically after the TTL.
+
+This functionality is supported by the lease mechanism. The readonly lease_id field is used to identify the lease of the property.
+
+### Examples of setting TTL
+
+```shell
+$ bydbctl property apply -f - <<EOF
+metadata:
+  container:
+    group: sw
+    name: ui_template
```

Comment:
Please replace all `ui_template` by `temp_data`. This could mislead people for TTL applied on the `ui_template`, which is not.

Reply:
Done